.coafile

```diff
@@ -1,7 +1,7 @@
 [default]
 bears = LineLengthBear, SpaceConsistencyBear
 files = **.js, **.css, **.html, **.yml, **.json
-ignore = **.min.js, **.min.css, tweetheatmap/public/lib/**, webtweets/js/bootstrap/**, webtweets/css/bootstrap.css, js/typeahead.bundle.js, webtweets/js/bootstrap.js
+ignore = **.min.js, **.min.css, tweetheatmap/public/lib/**, webtweets/js/bootstrap/**, webtweets/css/bootstrap.css, js/typeahead.bundle.js, webtweets/js/bootstrap.js, **/node_modules/**, **/index_bundle.js, WorldMoodTracker/index.html
```

Comment:
index.html and js files written by you should not be ignored by coala bears i guess.